scripts/tools/byOrganisationId/RegisterSyncServiceByOrgId.php

```diff
@@ -91,6 +91,9 @@ protected function createOrganisationIdProperty()
             WidgetRdf::PROPERTY_WIDGET => WidgetDefinitions::PROPERTY_TEXTBOX,
             TaoOntology::PROPERTY_GUI_ORDER => 1,
         ));
+
+        $property->setPropertiesValues([ValidationRuleRegistry::PROPERTY_VALIDATION_RULE => 'unique']);
```

Comment:
The script `SetupCentralServer` is called only once, and even if script `RegisterSyncServiceByOrgId` would be called again, it wouldn't add this `unique` validatator to orgid property. I think updater script should verify that the property is there and both validation rules are present

Reply:
removed